tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -182,6 +180,39 @@ export class GraphExecutor implements FunctionExecutor {
         this.graph, this.weightMap, executionInfo);
   }
 
+  private cloneAndKeepTensor(tensor: Tensor) {
+    if (tensor == null) {
+      return null;
+    }
+    const clone = tensor.clone();
+    // Keep the clone because`model.execute()` may be called within
+    // a `tidy()`, but the user may inspect these tensors after the
+    // tidy.
+    keep(clone);
+    return clone;
+  }
+
+  private cloneTensorMap(tensorsMap: NamedTensorsMap): NamedTensorsMap {
+    return Object.fromEntries(
+        Object.entries(tensorsMap).map(([name, tensorsList]) => {
+          return [
+            name, tensorsList.map(tensor => {
+              return this.cloneAndKeepTensor(tensor);
+            })
+          ];
+        }));
+  }
+
+  private cloneTensorList(
+      tensors: Tensor[], nodeName: string, tensorsMap: NamedTensorsMap) {
```

Comment:
```suggestion
      tensors?: Tensor[], nodeName?: string, tensorsMap?: NamedTensorsMap) {
```
We'd like to enable strictNullChecks in the future, so if you don't accept the nit that changes this function to return a list of tensors instead of mutating `tensorsMap`, please mark all these arguments as optional.

Reply:
return a list of tensors